internal/ipc/peerauth_unix.go

```
@@ -0,0 +1,122 @@
+		return c, nil
+	}
+}
+
+func (l *validatingListener) Close() error   { return l.inner.Close() }
+func (l *validatingListener) Addr() net.Addr { return l.inner.Addr() }
+
+// selfUID returns the current process's effective UID. Used by the
+// server to auto-populate the allowlist when the operator has not
+// configured any UIDs but the mode still requires one (unmanaged
+// operators occasionally set socket_mode=0666 for local testing).
+func selfUID() uint32 {
```

golangci-lint (2.12.2)

[error] func selfUID is unused

(unused)